scss/_patterns_equal-height-row.scss

```diff
@@ -15,21 +17,24 @@
       grid-row: span 4;
       grid-template-rows: subgrid;
       margin-bottom: $spv--small;
-      margin-top: -1px;
       position: relative;
 
       @media screen and ($breakpoint-small <= width < $breakpoint-large) {
         grid-column: span $grid-columns-medium;
         grid-template-columns: subgrid;
 
-        // for medium screen, each column item will take half of the available cols from the parent grid
+        // At medium size, each column item will take half of the available
+        // cols from the parent grid
         .p-equal-height-row__item {
           grid-column: span calc($grid-columns-medium / 2);
         }
 
-        // for medium screen, position the first column item on the left of the grid
+        // At medium size, position the first column item on the left of the
+        // grid
         .p-equal-height-row__item:first-child {
-          grid-row: span 100; // this needs to be sufficiently large so remaining column items won't get stretched
+          // This needs to be sufficiently large so remaining column items won't
+          // get stretched
+          grid-row: span 100;
```

Comment:
Nit: There's a bit of weirdness here with single-sentence comments being split into multiple single-line comments. I'm guessing a formatter did this.

If these sentences have to be multiple lines for readability they should use multi-line comments